swarm/pss/pssapi.go

```diff
@@ -62,16 +75,16 @@ func (pssapi *API) Receive(ctx context.Context, topic whisper.TopicType) (*rpc.S
 //
 // The method will pass on the error received from pss. It will fail if no symmetric key for the Pss peer has been added, or if the key has expired
 // The addresslength parameter decides how many bytes of the address to reveal in transit. -1 equals showing all. 0 means show none.
-func (pssapi *API) SendSym(topic whisper.TopicType, msg APIMsg, addresslength int8) error {
-	return pssapi.Pss.SendSym(msg.Addr, topic, msg.Msg, int(addresslength))
+func (pssapi *API) SendSym(key string, topic whisper.TopicType, msg []byte) error {
+	return pssapi.Pss.SendSym(key, topic, msg)
 }
 
 // Sends the message wrapped in APIMsg through pss using symmetric encryption
 //
 // The method will pass on the error received from pss. It will fail if no public key for the Pss peer has been added
 // The addresslength parameter decides how many bytes of the address to reveal it transit. -1 equals showing all. 0 means show none.
-func (pssapi *API) SendAsym(topic whisper.TopicType, msg APIMsg, addresslength int8) error {
-	return pssapi.Pss.SendAsym(msg.Addr, topic, msg.Msg, int(addresslength))
+func (pssapi *API) SendAsym(key string, topic whisper.TopicType, msg []byte) error {
```

Comment:
This method seems unnecessary since it is passed with the same parameters to the same method of `Pss` object. 
But since its an embedded type, that method is already defined on the API